### ISSUE TITLE
Don't report trivial attr readers when method name doesn't match the variable's name

### ISSUE THREAD

**Issue by gnapse** [rubocop/rubocop#408]
Regarding the rule that states that one should favor using `attr_reader` instead of defining the reader method manually, it triggers erroneously for the following scenario:

``` ruby
def current_command
  @command
end
```

Clearly in this case, even when the reader method is trivial, its name and the name of the variable being read are not the same, thus `attr_reader` is of no use here.
